src/pages/signin/ChangeExpensifyLoginLink.js

```diff
@@ -27,28 +28,31 @@ const defaultProps = {
     },
 };
 
-const ChangeExpensifyLoginLink = props => (
-    <View style={[styles.changeExpensifyLoginLinkContainer, styles.mt3]}>
-        <Text>
-            {props.translate('common.not')}
-            &nbsp;
-            {Str.isSMSLogin(props.credentials.login)
-                ? props.toLocalPhone(Str.removeSMSDomain(props.credentials.login))
-                : Str.removeSMSDomain(props.credentials.login)}
-            {'? '}
-        </Text>
-        <TouchableOpacity
-            style={[styles.link]}
-            onPress={Session.clearSignInData}
-            underlayColor={themeColors.componentBG}
-        >
-            <Text style={[styles.link]}>
-                {props.translate('common.goBack')}
-                {'.'}
+const ChangeExpensifyLoginLink = (props) => {
+    const login = lodashGet(props.credentials, 'login', '');
```

Comment:
question: Is this change still necessary? If the `login` is not in `credentials` and we are here then this is a bug.

Reply:
I think it's better to have a fallback for `credentials.login` to prevent crashing the app. In this case, the user would still be redirected to the username step and we would display an error message.

Reply:
**counterpoint:** If the app must crash we have an error boundary for it to hit. It's better to do that than to show content that requires a prop. It might even be _better_ to let the error boundary get hit as it alerts us of a problem by logging (instead of a confusing situation like rendering empty strings instead of required content).

**thought:** maybe the error boundary should also do `ENSURE_BUGBOT` so we can know about and fix stuff right away when it crashes instead of showing broken pages with required variables that we have gave defaults because we were afraid of crashes.

Reply:
Fair point. I reverted these changes.